classes/class-twentytwenty-customize.php

```diff
@@ -67,11 +67,71 @@ public static function register( $wp_customize ) {
 				)
 			);
 
+			/**
+			 * Implementation for the accent color.
+			 * This is different to all other color options because of the accessibility enhancements.
+			 * The control is a hue-only colorpicker, and there is a separate setting that holds values
+			 * for other colors calculated based on the selected hue and various background-colors on the page.
+			 *
+			 * @since 1.0.0
+			 */
+
+			// Add the setting for the hue colorpicker.
+			$wp_customize->add_setting(
+				'accent_hue',
+				array(
+					'default'           => 344,
+					'type'              => 'theme_mod',
+					'sanitize_callback' => 'absint',
+					'transport'         => 'postMessage',
+				)
+			);
+
+			// Add setting to hold colors derived from the accent hue.
+			$wp_customize->add_setting(
+				'accent_accessible_colors',
+				array(
+					'default'           => array(), // TODO: We need some sane defaults here.
+					'type'              => 'theme_mod',
+					'transport'         => 'postMessage',
+					'sanitize_callback' => function( $value ) { // TODO: This should probably be a normal mehod in this object instead of a closure.
+						$value = is_array( $value ) ? $value : array();
+						foreach ( $value as $context => $values ) {
+							$value[ $context ] = array(
+								'text' => ( isset( $value[ $context ]['text'] ) ) ? $value[ $context ]['text'] : '#000000',
+								'accent' => ( isset( $value[ $context ]['accent'] ) ) ? $value[ $context ]['accent'] : '#CD2653',
+							);
+						}
+						return $value;
+					},
+				)
+			);
+
+			// Add the hue-only colorpicker for the accent color.
+			$wp_customize->add_control(
+				new WP_Customize_Color_Control(
+					$wp_customize,
+					'accent_hue',
+					array(
+						'label'    => esc_html__( 'Accent Color Hue', 'twentytwenty' ),
+						'section'  => 'colors',
+						'settings' => 'accent_hue',
+						'priority' => 10,
+						'mode'     => 'hue',
+					)
+				)
+			);
+
 			/**
 			 * Colors.
 			*/
 			$accent_color_options = self::get_color_options();
 
+			/**
+			 * WIP: Disable these options 'cause we're doing the accessibility-color implementation above.
+			 */
+			$accent_color_options = array();
```

Comment:
This part here disables the settings added via the `get_color_options()` method. It's just here 'cause we need to decide what to do with the previous option.